.travis.yml

```diff
@@ -31,8 +31,6 @@ matrix:
           env: TOXENV=py35
         - python: 3.6
           env: TOXENV=py36
-        - python: 3.6
-          env: TOXENV=py36_ujson
```

Comment:
Ah, I have blindly ported this forward in the CPython 3.7 PR :grimacing: Will sync that one.